{"Launchpad":"https://bugs.launchpad.net/bugs/1482030","Id":"1482030","Gerrit":"https://review.openstack.org/209749","IdGerrit":"209749","IdCommit":"d14160e51bb800a0b9153464df42f34370d4b307","Files":["cinder/tests/unit/test_huawei_drivers.py","cinder/volume/drivers/huawei/huawei_driver.py","cinder/volume/drivers/huawei/rest_client.py"],"Lines_Inseted":35,"Lines_Deleted":41,"CommitParent":"351a432f592b42ad30e3c87b6520693b0c5df44a","Classification":"Bug","KeywordsTitle":"","Description":"","KeywordsCommit":"","Comments":""}